docs/en_US/Blog/HPOComparison.md

```diff
@@ -0,0 +1,323 @@
+# Hyperparameter Optimization Comparison
+*Posted by Anonymous Author*
+
+Comparison of Hyperparameter Optimization algorithms on several problems.
+
+Hyperparameter Optimization algorithms are list below:
+
+- Random Search
+- Grid Search
+- Evolution
+- Anneal
+- Metis
+- TPE
+- SMAC
+- HyperBand
+- BOHB
+
+All algorithms run in NNI local environment.
+
+## First Fake Function
+
+### Problem Description
+
+Firstly, one fake function is made to test the effectiveness in convex problem when hyperparameters are not independent.
+
+Fake Function:
+
+![](<http://latex.codecogs.com/gif.latex?Loss(x,y,z)=(x-30+0.1)^2\times(y-40+0.1)^2\times(x+z-50+0.1)^2>)
+
+### Search Space
+
+```json
+{
+  "x": {
+    "_type": "randint",
+    "_value": [100]
+  },
+  "y": {
+    "_type": "randint",
+    "_value": [100]
+  },
+  "z": {
+    "_type": "randint",
+    "_value": [100]
+  }
+}
+```
+
+The total search space is 1,000,000, the number of maximum trial to 1000. The time limitation is 24 hours.
+
+### Results
+
```

Comment:
1. Can you explain what is top1 and top5 loss? 
2. Is it necessary to list from top1 to top10 loss?   Considering if you only list top1, top3, top5, top10, do you lose any important information?

Reply:
1. My expression here (top1 and top5 loss) is ambiguous. I mean the top 10 of the least loss by by `Top10 loss`.
2. I don't know is there a better way to list information in multiple experiments. Maybe it's not necessary to do such thing. If we only list top1, top3, top5, top10, we can lose some information.

Reply:
I still do not understand the top10 loss,  is it the average of most smallest 10 loss values?

Reply:
what kind of information is lost if you only list  top1, top3, top5, top10 loss?

Reply:
Top10 loss I mean here is exact the smallest 10 loss values.

Reply:
The expression Top10 here is not appropriate and I will consider change it in other words.

Reply:
so the 'top10' column is the 10th smallest loss value, and 'top9' column is the 9th smallest loss value?

Reply:
Yes, that's what I want to express.

Reply:
Ok, as you said the name 'top x loss' here is confusing.
It is not necessary to list all 10 values here,  suggest list only best loss,  average of best 5 losses and average of best 10 losses.

Reply:
Thank you for your advice.